### DISCUSSION TITLE
Stream from both cameras

### DISCUSSION THREAD

**Discussion by andrewconlan** [HaishinKit/HaishinKit.swift#1058] [Q&A] [answered]
Hi trying to figure out if it's possible to create a stream that uses both the front and back cameras at the same time, so for example the selfie camera is added as a little window on top of the main video being recorded?

**Comment by shogo4405** [comment, marked as answer]
HaishinKit has no feature.

**Comment by shogo4405** [comment]
This feature is available now!
https://github.com/shogo4405/HaishinKit.swift/releases/tag/1.4.0
